department-of-veterans-affairs/vets-website
# Pension 70403 mvp total net worth

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Financial information' chapter to collect more information

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Financial Information' chapter
4. Continue to the 'Income and assets' page
5. Verify that the 'What are assets?' alert is collapsed
6. Confirm that the alert expands to reveal more information when you click the item
7. Confirm that the space button collapses the alert
8. Select 'Yes' and verify that the 'Submit Request Form' alert becomes visible
9. Select 'No' and verify that the alert is hidden
10. Verify the layout and content
11. Verify that the form data is saved to the redux store and submitted to the api

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70403)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Income and assets | Income and assets (Expanded)  | Income and assets (Yes)
| ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_financial-information_total-net-worth](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/87c2b5b1-8398-4557-ad68-8e38dfa42d4e)|![localhost_3001_pension_application_527EZ_financial-information_total-net-worth (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/eeb9f114-9865-444d-bbda-bed5073e3822)|![localhost_3001_pension_application_527EZ_financial-information_total-net-worth (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/38981947-4d55-4508-9d8f-89c249076d73)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70403)

### Quality Assurance & Testing

### Error Handling


### Authentication
